projects/plutus/inc/ltc_afe.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+// driver for LTC6804-1 AFE chip
+// requires GPIO, Interrupts and Soft Timers to be initialized
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include "gpio.h"
+#include "spi.h"
+#include "status.h"
+
+// static_assert shim macro since we aren't using C11
```

Comment:
I mean, in our ``Makefile`` platform rules, we seem to be using C99? I'd be happy to bump it to C11 to get rid of the macro and switch to the ``static_assert`` defined in ``<assert.h>``.

**./platform/stm32f0xx/platform.mk**:

```Makefile
 # Build flags for the device
 CDEFINES := USE_STDPERIPH_DRIVER STM32F072
 CFLAGS := -Wall -Werror -g3 -Os -std=c99 -Wno-unused-variable -pedantic \
           -ffunction-sections -fdata-sections -fno-builtin -flto \
           --specs=nosys.specs --specs=nano.specs \
           $(ARCH_CLAGS) $(addprefix -D,$(CDEFINES))
```

**./platform/x86/platform.mk**:

```Makefile
CFLAGS := $(CSFLAGS) -std=gnu99 -Wall -Werror -Wno-unused-variable -pedantic \
           -ffunction-sections -fdata-sections \
           -Wl,-Map=$(BIN_DIR)/$(PROJECT).map \
           $(ARCH_CLAGS) $(addprefix -D,$(CDEFINES))
```

The diff which added the ``-std=`` flag seems to be attached to #9. I assume you had a rationale at the time?

```diff
diff --git a/platform/stm32f0xx/platform.mk b/platform/stm32f0xx/platform.mk
index 92eb657..06de519 100644
--- a/platform/stm32f0xx/platform.mk
+++ b/platform/stm32f0xx/platform.mk
@@ -20,7 +20,7 @@ LDSCRIPT := $(PLATFORM_DIR)/ldscripts

 # Build flags for the device
 CDEFINES := USE_STDPERIPH_DRIVER STM32F072
-CFLAGS := -Wall -Werror -g -Os -Wno-unused-variable -pedantic \
+CFLAGS := -Wall -Werror -g -Os -std=c99 -Wno-unused-variable -pedantic \
           -ffunction-sections -fdata-sections -fno-builtin -flto \
           --specs=nosys.specs --specs=nano.specs \
           $(ARCH_CLAGS) $(addprefix -D,$(CDEFINES))
```